transport/http/config.go

```diff
@@ -133,13 +134,28 @@ func (ts *transportSpec) buildInbound(ic *InboundConfig, t transport.Transport,
 // 	      http:
 // 	        url: "http://127.0.0.1:80/"
 type OutboundConfig struct {
+	config.PeerListConfig
```

Comment:
You should mention in the OutboundConfig docs how `url` is interpreted if peer list is specified (possibly with an example)